modules/client-configuration/pages/supported-features.adoc

```diff
@@ -41,22 +41,22 @@ The operating system you run on a client is supported by the organization that s
 | Salt Clients
 
 | {sle} 15
-| {x86_64}, {ppc}, {ibmz}, ARM
+| {x86_64}, {ppc64le}, {ibmz}, ARM
 | {check}
 | {check}
 
 | {sle} 12
-| {x86_64}, {ppc}, {ibmz}, ARM
+| {x86_64}, {ppc64le}, {ibmz}, ARM
 | {check}
 | {check}
 
 | {sle} 11
-| {x86}, {x86_64}, {ipf}, IBM {ppc}, {ibmz}
+| {x86}, {x86_64}, {ipf}, IBM {ppc64le}, {ibmz}
 | {check}
 | {check}
 
 | {sles} for SAP
-| {x86_64}, {ppc}
+| {x86_64}, {ppc64le}
```

Comment:
This does not make any like without a version. It's not "SLES for SAP" but "SLES for SAP 12" and "SLES for SAP 15". And it's the same as for the normal SLES. It probably makes sense to duplicate this column and add the version number.

Reply:
Yeah, I wondered about that. I'll duplicate it.